internal/github/pr.go

```diff
@@ -148,3 +164,45 @@ func PRHasLabel(pr *github.PullRequest, label string) bool {
 
 	return false
 }
+
+func (ph *PRHelperImpl) GetUser(ctx context.Context, commit *object.Commit) (*github.User, error) {
```

Comment:
Not sure `PRHelper` is the best type for this service. Maybe there should be a new dedicated interface? `UserHelper`?

Reply:
Fixed.